mono/mini/exceptions-amd64.c

```diff
@@ -925,16 +925,24 @@ mono_amd64_get_exception_trampolines (gboolean aot)
 	MonoTrampInfo *info;
 	GSList *tramps = NULL;
 
+	// FIXME more in the macro
+#define TRAMPOLINE(name) (info->jit_icall_info = &mono_jit_icall_info.mono_ ## name)
+
```

Comment:
These macros are not needed.

Reply:
I'd like to expand them to have "every" line per trampoline.
I had that at first, but something wasn't working so I dialed back.
But ok either way.

Reply:
I remove all the `#define TRAMPOLINE`. There were several and they varied some. I'd like to put them back and have them do more, another time.